llvm/include/llvm/Analysis/ScalarEvolutionPatternMatch.h

```diff
@@ -0,0 +1,45 @@
+//===- ScalarEvolutionPatternMatch.h - Match on SCEVs -----------*- C++ -*-===//
```

Comment:
Per recent changes to guidelines, you can drop the file name and the emacs marker.

Reply:
Replaced with just  ```/===----------------------------------------------------------------------===//`